### PR TITLE
#6 Updates the datacite deposit to include the correct format of SeriesInformation and addition of RelatedItems.

### LINKED ISSUE #4: Mapping in Janeway to Datacite
The editors from ZIF were importing some metadata via DOI in Citavi and reported errors. Seems like the display in Citavi depends on the source of the title data if you use the picker (addon for the browser). The picker is a tool, you can click on the small symbol behind the DOI to import the title quickly into you Citavi project.

In JSON there are no separate objects for the journal metadata, everything is in the "description" field. Therefore, if the data source is Datacite, the metadata of the journal will be imported into the "Title supplement" field, see JSON File: https://api.datacite.org/application/vnd.datacite.datacite+json/10.48694/zif.3491

This is what it looks like in Citavi:
metadata_import

I think this is a mapping issue in the Datacite plugin, since the relevant metadata is showing up in the wrong field (e.g. periodical, volume and issue number in "description").

Entry in DataCite Commons: https://commons.datacite.org/doi.org/10.48694/zif.3491

Another odd observation: For some articles the source is not DataCite but the homepage. In this case the metadata is displayed correctly, except for collaborators. I don’t really understand what is triggering the source selection in the bibliographic management software:
metadata_import_2

We also did a test run with other programs. Maybe this documentation is helpful:
[report_ZIF_metadata.pdf](https://github.com/BirkbeckCTP/datacite/files/11464701/report_ZIF_metadata.pdf)

### LINKED ISSUE #6: Mapping in Janeway to Datacite (Datacite Metadata Schema)
We suggest a change to the plugin to better integrate the journal metadata. Currently, there is a mapping issue where the relevant data (name of the journal, volume, issue) is not displayed correctly in some reference management programs like Citavi. Some information is also missing for certain export formats (like bibtex, i.e. see https://commons.datacite.org/doi.org/10.48694/zif.3491)

[The new draft for version 4.5](https://datacite-metadata-schema.readthedocs.io/en/4.5_draft/guidance/related_item_guide.html#use-case-providing-publication-information-for-journal-articles-book-chapters-and-more) of the Datacite Metadata Schema suggests the "relatedItems" property to specify the metadata of the journal. The current schema version 4.4 already provides this way of keeping structured series information and has been partially adapted in this plugin. The complete information is currently in the "SeriesInformation" field, but not in the DataCite recommended format.

We believe that the journal metadata can be in both fields. We suggest the following:

1. [utils.py](https://github.com/BirkbeckCTP/datacite/blob/master/utils.py) line  11 et seq.: Change information in the "SeriesInformation" porperty to the recommended format from Datacite, see https://support.datacite.org/docs/field-descriptions-for-form#descriptions: _series title, followed by comma and optional volume(issue), firstpage-lastpage_

2. [utils.py](https://github.com/BirkbeckCTP/datacite/blob/master/utils.py) line 91 et seq.: Add code to include information in the "relatedItems" property, see https://support.datacite.org/docs/field-descriptions-for-form#related-items:
- Related Item Type: "Journal"
- Relation Type: "Is published in"
- Related Item Identifier: "{ISSN of the journal}"
- Related Identifier Type: "ISSN"
- Volume
- Issue
- Last Page
- First Page 
- You can compare the RelatedItem property with an example for XML and JSON [here](https://datacite-metadata-schema.readthedocs.io/en/4.5_draft/guidance/related_item_guide.html#example-journal-article-in-a-journal-with-an-issn)

Also see https://github.com/BirkbeckCTP/datacite/issues/4

### PR DESCRIPTION
**Pull request by ajrbyers**
Closes #4 and Closes #6

### CODE REVIEW

**Review by mauromsl — APPROVED** [review]

**Review by joemull — CHANGES_REQUESTED** [review]
Haven't been able to review this fully yet because of some server access issues, but I will raise those in standup and in the meantime point out what might be a missing piece of data in the string formatting. See comment inline.

File: utils.py [hunk 1]
```diff
@@ -12,12 +12,17 @@ def prep_data(article, doi, event=None):
     series_information = article.journal.name
 
     if article.issue:
-        series_information = "{} Volume {} Issue {} {}".format(
+        series_information = "{}, {}({})".format(
             article.journal.name,
             article.issue.volume,
             article.issue.issue,
             article.issue.date.year,
         )
```
[the remark this answers is not in the document]
Comment (ajrbyers) [reply utils.py:19]: Thanks Joe, actually its the last line of the `format` that has erroneously been left in place. I've added a commit to remove it.

**Review by joemull — APPROVED** [review]
Resolved the server issue and was able to complete review

**Comment by mattiker** [comment]
Hello, I hope this is the right place for questions or suggestions related to this issue 😅
I think the changes are not live yet. Can you perhaps already say when or with which version the changes will be adopted?

I also have a small suggestion for improving the "formats" lines: https://github.com/BirkbeckCTP/datacite/blob/657fa6200ea96374aad691fd62fbb0628f9458ed/utils.py#L30-L34

Datacite advises to use file extension or MIME type: https://datacite-metadata-schema.readthedocs.io/en/4.5/properties/format/#
So according to the [IANA media types](https://www.iana.org/assignments/media-types/media-types.xhtml), it should be:

- application/pdf
- application/xml

**Comment by ajrbyers** [comment]
I've opened another PR for this here: https://github.com/BirkbeckCTP/datacite/pull/10

**Pull request merged**